package.json

```diff
@@ -53,6 +53,7 @@
     "decanter": "^7.0.0-beta.1",
     "glob": "^7.1.7",
     "prop-types": "^15.7.2",
+    "react-aria-modal": "^4.0.0",
```

Comment:
This should be moved out of the dependencies and into optionalDependencies or peerDependencies and devDependencies as it isn't required.